awx/main/tests/functional/test_tasks.py

```diff
@@ -30,7 +30,7 @@ class TestDependentInventoryUpdate:
     def test_dependent_inventory_updates_is_called(self, scm_inventory_source, scm_revision_file):
         task = RunProjectUpdate()
         task.revision_path = scm_revision_file
-        proj_update = ProjectUpdate.objects.create(project=scm_inventory_source.source_project)
+        proj_update = scm_inventory_source.source_project.create_project_update()
```

Comment:
We can't do `objects.create()` because that won't set `local_path`. It has to be created via the conical means of creating it _from_ the project.